client/components/category-select/select.template.html.slim

```diff
@@ -0,0 +1,25 @@
+/ category selection
+.form-group
+  label ng-if='!noLabel && !multiple' for='{{ prefix }}-category-select-single' Category
+  label ng-if='!noLabel && multiple' for='{{ prefix }}-category-select-multiple' Category
+
+  / check box to choose whether to create a new category or not
+  label.new-switch.pull-right ng-if='createNew'
+    input type='checkbox' ng-model='config.newCategory' ng-disabled='!categoryChoices.length'
+    '
+    ' create a new category
+
+  / Single dropdown menu to select an existing category
+  ui-select id='{{ prefix }}-category-select-single' ng-if='!multiple && !config.newCategory && modelProperty' ng-model='modelObject[modelProperty]' reset-search-input=true
+    ui-select-match placeholder='{{ getPlaceholder() }}' allow-clear='true' {{ $select.selected.name }}
+    ui-select-choices repeat='category[extract] as category in categoryChoices | filter: $select.search' refresh='fetchCategoryChoices($select.search)' refresh-delay='250'
```

Comment:
The `filter` is probably no longer necessary (in both single and multiple versions of the `ui-select`).